### PR TITLE
Update TSC Attendance Governance 

### PR DESCRIPTION
**Pull request by GoldenBit0**
Per discussions in recent PGB, TSC, and Core Dev calls, this PR contains the updated TSC attendance governance for review by the PGB.

If the PGB approves, 'Do Not Merge' will be removed and the maintainers can proceed with standard PR approval process.

# Description

Updated TSC attendance governance to include what happens if x meetings are missed, process for removal, etc.

## Motivation and Context

This change is to improve clarify for TSC attendance policy and process.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Request to be added as a Code Owner/Maintainer

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I commit to abide by the Responsibilities of Code Owners/Maintainers.

### CODE REVIEW

File: docs/governance/governance.md [hunk 1]
```diff
@@ -30,9 +30,18 @@ A quorum of two-thirds of the TSC members can conduct any vote required of the T
...
 
-The TSC's members are elected annually in September/October, concluding on the last day of that month, with the first elections held in September, 2020. The period for nominations will be announced no less than 30 days prior to elections. The method and management of the nominating process and the elections will be communicated to the community by the various [channels](../../community/open-source-community.md#communications-connecting-with-each-other-directly) in that timeframe
+#### Removal
+
+A TSC member is eligible to lose their seat upon missing two consecutive TSC meetings or three total during a period between elections, or for violating the Code of Conduct.
+
+If a TSC member does not comply with the attendance policy, a simple majority vote for removal is conducted by Oasis for the remaining TSC members that are not being considered for removal to vote.
```
Comment (tasdienes) [inline docs/governance/governance.md:39]: Is it necessary for the vote to be "conducted by Oasis"?

Comment (Therecanbeonlyone1969) [inline docs/governance/governance.md:39]:
```suggestion
If a TSC member does not comply with the attendance policy, a simple majority vote of the remaining TSC members that are not being considered for removal is sufficient to remove the TSC member.
```

**Review by Therecanbeonlyone1969 — CHANGES_REQUESTED** [review]

**Comment by GoldenBit0** [comment]
Need to update PR to match voting rights to PGB voting eligibility.

**Comment by GoldenBit0** [comment]
4/12/23 PGB Call: Simple majority unanimous approval for the updated TSC governance.

PR is ready for review/merge.

**Review by Therecanbeonlyone1969 — APPROVED** [review]
LGTM

**Review by ognjenkurtic — APPROVED** [review]

**Pull request merged**